OpenSim/Simulation/StatesTrajectory.h

```diff
@@ -44,20 +44,22 @@ class Model;
 // users are likely to be interested in a more general State container that
 // doesn't assume the states are sequential in time.
 
-/** A sequence of SimTK::State%s that can be saved to a plain-text OSTATES
- * file.  The trajectory can also be populated from such a file. You may obtain
- * a StatesTrajectory from a simulation, or other numerical methods whose task
- * is to produce a trajectory of states. Users can modify the trajectory by
- * appending states to it, but users cannot modify the individual states that
- * are already in the trajectory.
+// TODO See the bottom of this file for a class description to use once the
+// OSTATES file format is implemented.
+//
+/** This class holds a sequence of SimTK::State%s. You can obtain a
+ * StatesTrajectory during a simulation via the StatesTrajectoryReporter. You
+ * can also create a StatesTrajectory from a states storage (.sto) file (see
+ * createFromStatesStorage()). Users can modify a trajectory by appending
+ * states to it, but users cannot modify the individual states that are already
+ * in a trajectory.
  *
  * This class was introduced in OpenSim version 4.0, and enables scripting
  * (Python/MATLAB) and C++ users to postprocess their results with greater ease
- * and more versatility than with an Analysis.
+ * and flexibly than with an Analysis.
```

Comment:
I suggested "flexibility" (with greater ease and [greater] flexibility than…); I don't think "flexibly" makes sense (with… flexibly than…).